### DISCUSSION TITLE
missing locales

### DISCUSSION THREAD

**Discussion by Rakasch** [primefaces/community#449] [PrimeVue] [answered]
I just noticed my file upload component has stopped working. 
And the reason is that PrimeVue couldn't find the translation for the key "fileSizeTypes" in my locales. 
Upon checking, I found that it was recently added to the PrimeVue locales. Therefore, I obviously didn't have it in my local translations. 
We only noticed very late on that the component was no longer working properly. Since you only notice there is a problem when you want to upload something.

Is it intended for this to break the component if I don't have the translation in my locale? 
If yes, I would appreciate it if I could get a clear indication of information in the changelog, as I consider it literally a breaking change.

Another possibility would be to merge the local translation with the English default. Instead of just overwriting the whole locale.
This way there would always be a fallback value. 
You may also want to display a warning about missing translations.

Somehow the process here needs to be improved.

**Comment by tugcekucukoglu** [comment, marked as answer]
`fileSizeTypes` should not need to be added for backward compatibility. English transition is added by default for the next version.
